readme.md

```diff
@@ -101,7 +101,6 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
   - Desktop development with C++
 
 ### macOS:
-- Xcode Command Line Tools
 - [Homebrew](https://brew.sh)
```

Comment:
L150 could do with rewording. The way it's worded suggests there is another way.